CIP-XXXX/README.md

```diff
@@ -0,0 +1,146 @@
+
+- Registering a stake credential
+- Delegating stake to a pool
+- Delegating voting power to a DRep
+
+This approach results in many complex and convoluted certificate types, increasing complexity for developers and users. The number of certificate types would also grow combinatorially with new features, making it only harder to document, extend, and understand the certificates.
+
+By redesigning the certificate types with orthogonal certificates, we can:
+ - Make the system more intuitive for users and developers
+ - Simplify documentation by having self-describing, single-purpose certificates
+ - Make ledger implementations easier to maintain
+ - Avoid combinatorial growth of certificates with future features
+
+
+## Specification
+
+### Current Certificates CDDL
+The Conway era has multiple certificate types for stake related operations:
+```cddl
+certificate = [stake_registration
+               // stake_deregistration
+               // stake_delegation
+               // pool_registration
+               // pool_retirement
+               // reg_cert
+               // unreg_cert
+               // vote_deleg_cert
+               // stake_vote_deleg_cert
+               // stake_reg_deleg_cert
+               // vote_reg_deleg_cert
+               // stake_vote_reg_deleg_cert
+               // auth_committee_hot_cert
+               // resign_committee_cold_cert
+               // reg_drep_cert
+               // unreg_drep_cert
+               // update_drep_cert]
+
+stake_registration = (0, stake_credential)
+stake_deregistration = (1, stake_credential)
+stake_delegation = (2, stake_credential, pool_keyhash)
+reg_cert = (7, stake_credential, coin)
+unreg_cert = (8, stake_credential, coin)
+stake_vote_deleg_cert = (10, stake_credential, pool_keyhash, drep)
+stake_reg_deleg_cert = (11, stake_credential, pool_keyhash, coin)
+stake_vote_reg_deleg_cert = (13, stake_credential, pool_keyhash, drep, coin)
+...
+```
+
+### Proposed CDDL
+While the current certificates may save a few bytes in the transaction when doing multiple actions in a single transaction, it greatly complicates certificates for both users and implementers.
+
+Instead, we propose a much simpler certificate design with orthogonal certificates.
+```cddl
+certificate = pool_registration
+               / pool_retirement
+               / auth_committee_hot_cert
+               / resign_committee_cold_cert
+               / reg_drep_cert
+               / unreg_drep_cert
+               / update_drep_cert
+               / stake_registration_cert
+               / stake_deregistration_cert
+               / stake_delegation_cert
+               / vote_delegation_cert
+
+; New orthogonal certificates
+stake_registration_cert = (N, stake_credential, coin)
+stake_deregistration_cert = (N+1, stake_credential, coin)
+stake_delegation_cert = (N+2, stake_credential, pool_keyhash)
+vote_delegation_cert = (N+3, stake_credential, drep)
```

Comment:
For backward-compatibility reason, it would be necessary here to specify the new indices, and makes sure that they don't clash with the current ones. This way, one could _virtually_ still deserialise a compound certificate into two orthogonal ones; and ensure that we don't needlessly break all transaction builders once again.

Reply:
There is no need to break existing specification. All you trying to do is remove some certificates, while leaving others unchanged.
```suggestion
stake_registration_cert = (7, stake_credential, coin)
stake_deregistration_cert = (8, stake_credential, coin)
stake_delegation_cert = (2, stake_credential, pool_keyhash)
vote_delegation_cert = (9, stake_credential, drep)
```
Otherwise we would have to keep old certificates for one more era to avoid breakage, just to replace certificates with equivalent ones

Reply:
I am weighing in. Renaming (changing the tag) will require old certificates to be retained for a whole new era. So, it makes no sense to introduce new identical replacement certificates. Please adjust the CIP to retaon the old tags.

Actual names of cddl bindings are irrelevant and can be discussed at a any point, since they do not require a HF.